src/app/AppLayout/AppLayout.tsx

```diff
@@ -284,7 +283,7 @@ export const AppLayout: React.FC<AppLayoutProps> = ({ children }) => {
     () => [
       <FeatureFlag level={FeatureLevel.BETA} key={'language-preferences-feature-flag'}>
         <DropdownItem key={'language-preferences'} onClick={handleLanguagePref}>
-          <Trans t={t} components={{ Icon: <LanguageIcon /> }} i18nKey="AppLayout.USER_MENU.LANGUAGE_PREFERENCE" />
+          <Trans t={t} i18nKey="AppLayout.USER_MENU.LANGUAGE_PREFERENCE" />
```

Comment:
I'd like to keep this in some form, even if it gets redesigned. If the user opens the UI and cannot read the language it is currently set to, then having a language icon here can help them find the control they need to get a localization they do understand.

It's a bit of an academic point right now since the only localization is English anyway, but... thinking of the future :-)

Reply:
Sounds good! Now that you mentioned, it seems very useful indeed (i.e. been in that situation myself).

I added the icons now. Just that I need to add one for logout button too but I could not find any from `patternfly/react-icons` so I made one from Front-Awesome: https://fontawesome.com/icons/arrow-right-from-bracket?f=classic&s=solid

![image](https://github.com/user-attachments/assets/b254e77d-3c4f-47f3-9e4c-0092755d5e28)

Reply:
It should also work in dark theme.

![image](https://github.com/user-attachments/assets/ac702a90-b3ba-46c8-8716-d2864f4fb01a)